build-support/bin/native/bootstrap.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
This appears to be unrelated and on inspection it looks like it should interact OK with `build-support/bin/native/generate-bintray-manifest.sh`, which sources it, but it would be nice to separate the change.